src/test/msgr/test_msgr.cc

```diff
@@ -1775,16 +1775,21 @@ class SyntheticWorkload {
   DummyAuthClientServer dummy_auth;
 
  public:
-  static const unsigned max_in_flight = 64;
-  static const unsigned max_connections = 128;
+  unsigned max_in_flight;
```

Comment:
this member variable can, and should be marked `const`, i think.

Reply:
yes, already make it const.